libtransmission/peer-mgr.cc

```diff
@@ -2427,12 +2427,6 @@ namespace connect_helpers
 /* is this atom someone that we'd want to initiate a connection to? */
 [[nodiscard]] bool is_peer_candidate(tr_torrent const* tor, tr_peer_info const& peer_info, time_t const now)
 {
-    // have we already tried and failed to connect?
-    if (auto const conn = peer_info.is_connectable(); conn && !*conn)
-    {
```

Comment:
for clarity, should it be:

```
if !peer_info.is_connectable() || !peer_info.reconnect_interval_has_passed(now)
{
    return false;
}
```

Is there a way to be is_connectable but also in reconnect internal?

Reply:
AFAICT you are correct on this, but I'd rather not make things any more complicated. We keep it simple, and it's less likely to break.